app/assets/javascripts/application.js

```diff
@@ -27,6 +27,8 @@ window.addEventListener('load', () => {
             widthFixed: true,
             widgets: ['stickyHeaders', 'filter', 'zebra'],
         });
+        search = document.getElementsByClassName("tablesorter-filter-row tablesorter-ignoreRow");
+        search[0].removeAttribute("role");
```

Comment:
If we only need to change the first match, could we use `document.querySelector` instead?

```suggestion
        search = document.querySelector(".tablesorter-filter-row.tablesorter-ignoreRow");
        search?.removeAttribute("role");
```